pytensor/tensor/basic.py

```diff
@@ -185,6 +185,10 @@ def _as_tensor_numbers(x, name, ndim, dtype=None, **kwargs):
     return constant(x, name=name, ndim=ndim, dtype=dtype)
 
 
+def _as_tensor_numbers(x, name, ndim, dtype=None, **kwargs):
+    return constant(x, name=name, ndim=ndim, dtype=dtype)
```

Comment:
Are you using/needing this anywhere?

Reply:
I had added this in a previous commit, I don't think this is needed now